include/zmq.h

```diff
@@ -371,6 +374,18 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg, const char *property)
 #define ZMQ_VMCI_BUFFER_MAX_SIZE 87
 #define ZMQ_VMCI_CONNECT_TIMEOUT 88
 #define ZMQ_USE_FD 89
+#define ZMQ_PEER_EVENTS 90
+
+/*  getsockopt additional struct to recover peer events                       */
+typedef struct zmq_gso_peer_events_t
```

Comment:
I think the C++ style `struct zmq_gso_peer_events_t { ... };` should be preferred.

Reply:
I don't get it; are you arguing about the use of `typedef`?

Aside from not being a C++ style thing, the entire `zmq.h` has `typedef`s for each and every single of the structs.

Please clarify.

Reply:
Forget about this. This is the C zmq.h, not an internal C++ header. It must be kept as it is.